### PR TITLE
Add flags to concept model

### PR DESCRIPTION
**Pull request by leggsimon**
@laurieboyes @keirog

### CODE REVIEW

File: models/concept.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+export default (item, opts, { flags = {} }) => ({
 	type: 'concept',
 	id: item.id,
 	name: item.name,
 	items: item.items,
 	taxonomy: item.taxonomy,
 	url: item.url,
-	isFollowing: item.isFollowing
+	isFollowing: item.isFollowing,
+	flags: flags
```
Comment (keirog) [inline models/concept.js]: You could have that as just `flags` if you liked

**Comment by keirog** [comment]
👍  Nice

File: models/concept.js (hunk 1 above)
Comment (leggsimon) [inline models/concept.js]: Ah, I was following how it was done in `models/video.js` https://github.com/Financial-Times/n-section/blob/master/models/video.js#L6

**Comment by laurieboyes** [comment]
👍

**Pull request merged**